### PR TITLE
chore: code refactor and testing changes

### PR DESCRIPTION
**Pull request by domenicsim1**

### CODE REVIEW

**Review by benPearce1 — APPROVED** [review]
this looks good, feel free to merge to the other branch and I can fix up any issues

File: pkg/cmd/projectgroup/create/create.go [hunk 1]
```diff
@@ -153,3 +128,19 @@ func PromptMissing(opts *CreateOptions) error {
 
 	return nil
 }
+
+func AskName(ask question.Asker, messagePrefix string, resourceDescription string, value *string) error {
```
Comment (slewis74) [inline pkg/cmd/projectgroup/create/create.go:132]: we talked about having this in a shared package, will that be in a separate PR?

**Review by slewis74 — APPROVED** [review]
Very nice. Couple of whitespace fixes as suggestions

File: pkg/cmd/projectgroup/create/create.go (hunk 1 above)
Comment (domenicsim1) [reply pkg/cmd/projectgroup/create/create.go:132]: My goal here we to get everything ready @benPearce1 so he could write tests and finish off project create, as a result of forgot to move it 😅 . I will move it, and fix the small nits but there will be another PR after that updates all the commands to use the common shared questions.
